### PR TITLE
feat:check and output invalid users

### PR DESCRIPTION
**Pull request by cunla**
This should:
- Ignore invalid users in the `org.tfvars` file.
- Output as part of the `apply` command the invalid users (which should be removed afterwards).

### CODE REVIEW

File: terraform/resources-org.tf [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # GitHub Membership Resource
 # https://registry.terraform.io/providers/integrations/github/latest/docs/resources/membership
+data "github_users" "users" {
+  usernames = setunion(var.admins, var.members)
```
[the remark this answers is not in the document]
Comment (cunla) [reply terraform/resources-org.tf:4]: not sure I got what you meant. I removed `locals.admins`/`local.members` now..

[the remark this answers is not in the document]
Comment (tim-schilling) [reply terraform/resources-org.tf:4]: Ah nevermind. It was redefining admins and members.

File: terraform/locals.tf [hunk 2]
```diff
@@ -3,21 +3,6 @@
 
 locals {
 
-  admins = {
-    for user in var.admins : user => "admin"
-  }
-
-  branch_protections = {
```
Comment (ryancheley) [inline terraform/locals.tf:10]: it looks like we’re removing branch protection entirely. Is that expected?

Comment (cunla) [reply terraform/locals.tf:10]: it was never really used directly, as can be seen in the `tf plan` comment.
However, I fixed the `plan` gha to run on changes to `.tf` files and also removed the `enable_branch_protection` from every mention.

**Review by tim-schilling — APPROVED** [review]
Looks good to squash and merge. Thanks Daniel!

**Pull request merged**